requirements.yml

```diff
@@ -18,20 +17,20 @@ mysql:
     as: DB
     master: true
 
+workers:
+  - name: sidekiq
+    command: bin/sidekiq
```

Comment:
You can switch this to an array to resolve the requirements.yml warning, but you'll need to update the Dockerfile to remove the ENTRYPOINT, changing
```
ENTRYPOINT ["bin/unicorn"]
CMD ["-c", "config/unicorn.rb"]
```
to 
```
CMD ["bin/unicorn", "-c", "config/unicorn.rb"]
```
(this is also required for the pre-deploy command to work as an array)